metadata.json

```diff
@@ -8,17 +8,13 @@
   "project_page": "http://github.com/puppetlabs/puppetlabs-mysql",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
-    {
-      "name": "puppetlabs/translate",
-      "version_requirement": ">= 1.0.0 < 2.0.0"
-    },
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 3.2.0 < 5.0.0"
     },
     {
-      "name": "puppet/staging",
-      "version_requirement": ">= 1.0.1 < 4.0.0"
+      "name": "puppetlabs/translate",
```

Comment:
the diff would be a tiny bit cleaner if this would still be on top of stdlib, but it doesn't really matter.